.github/workflows/test-chart-version-nightly-template.yaml

```diff
@@ -0,0 +1,72 @@
+# Basic test for maintenance Camunda 8 Helm charts.
+name: "Test - Chart Version Nightly - Template"
+
+on:
+  workflow_call:
+    inputs:
+      camunda-version:
+        description: Camunda version to test
+        required: true
+        type: string
+      scenarios:
+        description: Scenarios
+        required: true
+        type: string
+      case:
+        description: Type
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ inputs.camunda-version }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  validation:
+    name: Camunda ${{ inputs.camunda-version }} - Validation
+    uses: ./.github/workflows/chart-validate-template.yaml
+    with:
+      identifier: "${{ inputs.case }}-vald-${{ inputs.camunda-version }}"
+      camunda-helm-dir: "camunda-platform-${{ inputs.camunda-version }}"
+      camunda-helm-git-ref: "${{ github.event.pull_request.head.sha }}"
+  init:
+    needs: validation
+    runs-on: ubuntu-latest
+    env:
+      SCENARIOS_LIST: ${{ inputs.scenarios }}
+    outputs:
+      scenarios-formatted: ${{ steps.make-json-array.outputs.json_array }}
+    steps:
+      - name: Convert string to JSON array and set as output
+        id: make-json-array
+        run: |
+          # For example, this converts the SCENARIOS_LIST ("multitenancy, oidc") to ["multitenancy","oidc"]
+          # This is done because it is not possible to pass arrays as input to a workflow. Therefore,
+          # a string is passed and converted to a json array
+          json=$(echo "$SCENARIOS_LIST" | jq -R -c 'split(",")')
+          echo "json_array=$json" | tee -a $GITHUB_OUTPUT
+  integration-scenario:
```

Comment:
Following the same pattern in other workflows, let's just call it `integration:` and add a new line before it.

Reply:
done